secretBatchRetrievalConnector/.taskkey

```diff
@@ -1 +1 @@
-a9061958-8d36-403e-9db7-a7fd83a43f24
\ No newline at end of file
+0df57e9a-069c-4ea2-895c-47f71aafab2f
```

Comment:
are we adding the api key?

Reply:
This is not an api key, it is taskkey like version number maintained by azure whenever we make changes to task(extension code)